setuptools/build_meta.py

```diff
@@ -402,10 +404,11 @@ def build_wheel(
     ):
         with suppress_known_deprecation():
             return self._build_with_temp_dir(
-                ['bdist_wheel', *self._arbitrary_args(config_settings)],
+                ['bdist_wheel'],
                 '.whl',
                 wheel_directory,
                 config_settings,
+                True,
```

Comment:
We probably can avoid the conditional all together this way:

```suggestion
                self._arbitrary_args(config_settings),
```

(untested suggestion)

Reply:
applied as suggested